### ISSUE TITLE
Фильтрация отображаемых проектов при создании задачи в боте

### ISSUE THREAD

**Issue by gr1ff0n** [nixys/nxs-support-bot#10]
Возможно ли отфильтровать список доступных проектов при создании задачи через бота?
Отфильтровать можно по признаку включенного функционала "Задачи" у проекта.

**Comment by AndreiMuvila** [comment]
Нет, такого функционала в боте нет. Сейчас в меню выводятся проекты, доступные текущему пользователю. 

Можете пояснить, в каких случаях и для чего будут использоваться проекты, в которых нельзя создавать задачи?

**Comment by gr1ff0n** [comment]
В нашем случае проект ИТ является родительским проектом, все задачи по-умолчанию приходят в подпроект нашего отдела. Задачи, отдаваемые подрядчикам, переносим в соответствующий подпроект либо связываем их с новыми задачами в данных подпроектах, где соответствующие подрядчики их видят. Поэтому в родительском проекте задачи, по-умолчанию, отсутствуют.

**Comment by AndreiMuvila** [comment]
То есть вы хотите чтобы этот родительский проект ИТ не светился в выдаче?

**Comment by gr1ff0n** [comment]
Да, всё верно вы поняли

**Comment by AndreiMuvila** [comment]
Спасибо за хорошее предложение. Мы добавим его в бэклог.
